code-style/php/README.md

```diff
@@ -0,0 +1,169 @@
+# PHP Coding Standards
+![](php-logo.png)
+
+Despite still being one of the most used languages to build web applications, PHP doesn't have the best reputation. There are a number of frameworks we use that are built with PHP. In order to avoid PHP's reputation of messy codebases, we should utilize [PHP_CodeSniffer](phpcs) to detect violations of a defined coding standards.
+
+### What Standard Should I use?
+
+Many frameworks have their own standards declared. We should use the framework's defined standard when possible. For instance, when developing a WordPress theme, you should utilize the [WordPress Standard](wp). If there is not a standard defined, or you are not using a framework, you should use the [PSR-2 Standard](psr-2).  Some standards are:
+
+- [PSR-2 / Default](psr-2)
+- [WordPress](wp)
+- [Symfony](symfony)
+- [Drupal](drupal)
+
+### Enforce my standard with PHPCS
+
+[PHP_CodeSniffer](phpcs) allows us to lint our PHP. To install PHPCS:
+
+1. Install PHP CodeSniffer:
+
+    `composer global require "squizlabs/php_codesniffer=*"`
+1. You need access to global composer installs. In your .zshrc file, ensure you have:
```

Comment:
I know that most (if not all) of us at Sparkbox use zsh, but for other other folks, should we also specify `.bashrc` as well?

Reply:
That seems like a reasonable suggestion to me :-D